ledger/acctonline.go

```diff
@@ -163,7 +191,7 @@ func (ao *onlineAccounts) close() {
 		ao.voters = nil
 	}
 
-	ao.baseAccounts.prune(0)
+	// ao.baseAccounts.prune(0)
```

Comment:
keep or remove this?

Reply:
same, will be done as a separate PR. Cannot reuse baseAccounts as is because the underlying data struct is different